sdk/servicebus/service-bus/test/internal/node/operationOptionsForAMQP.spec.ts

```diff
@@ -38,12 +39,14 @@ describe("OperationOptions reach getToken at `@azure/identity`", () => {
     func: Function,
     plugAfterMockingGetToken?: Function
   ) {
-    let getTokenIsInvoked = false;
     let verifiedOperationOptions = false;
     let actualOptions: OperationOptionsBase = {};
+    let getTokenInvocationDone: Function;
+    const waitUntilGetTokenIsInvoked = new Promise((resolve) => {
```

Comment:
`unittestUtils.ts` has `getPromiseResolverForTest` for this if you want to use it.

Reply:
I see.

Reply:
Got getPromiseResolverForTest